log-manager/src/main/java/io/airlift/log/Logging.java

```diff
@@ -237,5 +263,38 @@ public synchronized void configure(LoggingConfiguration config)
                 throw new UncheckedIOException(e);
             }
         }
+
+        exportJmxOnConfiguration(jmxExporterFuture, jmxExportCollector);
+    }
+
+    private static void exportJmxOnConfiguration(ListenableFuture<MBeanExporter> jmxExporterFuture, List<JmxExport> jmxExports)
```

Comment:
I'm pretty sure we need a way to unexport

Reply:
Added